docs/how-to-help-with-video-challenges.md

```diff
@@ -86,7 +86,7 @@ videoId: nVAaxZ34khk
 ```
 
 Next, access the YouTube video with that videoId. The URL for the video will be:
```

Comment:
```suggestion
Next, access the YouTube video with that `videoId`. The URL for the video will be:
```

Also, above this, the following appears:
> For example, in the following code from the header of a video challenge markdown file, the videoId is "nVAaxZ34khk". On GitHub, the information should be laid out in a table format.

It would be nice to add backticks around `videoId`.